### PR TITLE
CB-8396 - Read property "AutoHideSplashScreen" on android

### PR DESCRIPTION
**Pull request by tfitschen**

### CODE REVIEW

File: src/android/SplashScreen.java [hunk 1]
```diff
@@ -91,7 +91,9 @@ protected void pluginInitialize() {
 
         firstShow = false;
         loadSpinner();
-        showSplashScreen(true);
+
+        boolean hideAfterDelay = preferences.getBoolean("AutoHideSplashScreen", true);
```
Comment (agrieve) [inline src/android/SplashScreen.java]: The AutoHide preference is meant to hide without the use of a timer (e.g. hide on first paint after onload). Would be happy to accept a PR for this, but I don't think we should have a PR that just turns it off. For this, set a very large timeout.

**Comment by miqmago** [comment]
+1

**Comment by kentmw** [comment]
+1

**Comment by infil00p** [comment]
Does this actually work if you have a signficantly large application loading in the background? Or is this going to cause more white flashes?

**Comment by kentmw** [comment]
Good question, I haven't been able to test this. But it seems crazy that AutoHideSplashScreen isn't being read currently on the android side. Is someone able to test this patch? It also has conflicts.

**Comment by sencenan** [comment]
+1

**Comment by daserge** [comment]
@tfitschen, thanks for your contribution!
This has been included into #74, could you please close the PR?

**Pull request closed without merging**